internal/go/skycfg/proto_api.go

```diff
@@ -45,16 +45,19 @@ func NewProtoModule(registry ProtoRegistry) *ProtoModule {
 	mod := &ProtoModule{
 		Registry: registry,
 		attrs: starlark.StringDict{
+			"name":        starlark.NewBuiltin("proto.clone", fnProtoName),
 			"clear":        starlark.NewBuiltin("proto.clear", fnProtoClear),
 			"clone":        starlark.NewBuiltin("proto.clone", fnProtoClone),
 			"from_json":    starlark.NewBuiltin("proto.from_json", fnProtoFromJson),
 			"from_text":    starlark.NewBuiltin("proto.from_text", fnProtoFromText),
 			"from_yaml":    starlark.NewBuiltin("proto.from_yaml", fnProtoFromYaml),
+			"from_proto":   starlark.NewBuiltin("proto.from_proto", fnProtoFromProto),
 			"merge":        starlark.NewBuiltin("proto.merge", fnProtoMerge),
 			"set_defaults": starlark.NewBuiltin("proto.set_defaults", fnProtoSetDefaults),
 			"to_json":      starlark.NewBuiltin("proto.to_json", fnProtoToJson),
 			"to_text":      starlark.NewBuiltin("proto.to_text", fnProtoToText),
 			"to_yaml":      starlark.NewBuiltin("proto.to_yaml", fnProtoToYaml),
+			"to_proto":     starlark.NewBuiltin("proto.to_proto", fnProtoToProto),
```

Comment:
Let's add functions like `to_any`, which return the `google.protobuf.Any` directly. This way users don't have to deal with the internal details of the type URLs, and don't need to define their own marshal/unmarshal helpers.